### ISSUE TITLE
Feature Request: Method to temporarily disable ad-blocking.

### ISSUE THREAD

**Issue by magn2o** [TechnitiumSoftware/DnsServer#82]
It would be convenient if there was a quick and easy way to temporarily suspend ad-blocking for a set period of time (e.g. 5m, 30m, 1h, etc).

**Comment by ShreyasZare** [comment]
Thanks for the feedback. Will evaluate this feature and will let you know here when its ready.

**Comment by rlagmlah** [comment]
I also like this feature. Shopping ads are generally annoying, but sometimes useful when shopping.
I think it would be nice to be able to stop temporarily, like the competition project Pi-Hole.

**Comment by EHRETic** [comment]
If I might add something because I just got a "wife use case"... 😁
She wanted to see a replay on a TV channel, but the site was verifying static.adsafeprotected.com
Not reachable --> no video (and wife unhappy meaning complaining until I find a solution)

I had to look in source code to find the guilty one. It was of course feasible but now I have a permanent exception, which I really hate.

IF user could deactivate temporary one or more ad servers in a self service page (offered page would relate to source IP and blocked names logs maybe?), that would be a dream! 😉
